diff --git a/CMakeLists.txt b/CMakeLists.txt
index d51dfad..2b2e492 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,13 +16,6 @@ endif()
 set(VERSION "${VERSION}${VERSION_SUFFIX}")
 message("Version ${VERSION}")
 
-set(LLDB_PACKAGE $ENV{LLDB_PACKAGE} CACHE PATH "Zip archive containing LLDB files")
-if (LLDB_PACKAGE)
-    message("Using LLDB_PACKAGE=${LLDB_PACKAGE}")
-else()
-    message(FATAL_ERROR "LLDB_PACKAGE not set." )
-endif()
-
 set(CMAKE_CTEST_ARGUMENTS "--output-on-failure" CACHE INTERNAL "CTest arguments")
 
 if (CMAKE_SYSROOT)
@@ -125,15 +118,6 @@ configure_file(package.json ${CMAKE_CURRENT_BINARY_DIR}/package.json @ONLY)
 # We still need to resolve codelldb schema $ref's in it, but we'll do this during the build.
 file(COPY_FILE ${CMAKE_CURRENT_BINARY_DIR}/package.json ${CMAKE_CURRENT_BINARY_DIR}/package.pre.json)
 file(COPY_FILE ${CMAKE_CURRENT_SOURCE_DIR}/package-lock.json ${CMAKE_CURRENT_BINARY_DIR}/package-lock.json)
-# Install node_modules
-execute_process(
-    COMMAND ${NPM} install
-    WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
-    RESULT_VARIABLE Result
-)
-if (NOT ${Result} EQUAL 0)
-    message(FATAL_ERROR "npm install failed: ${Result}")
-endif()
 # So we can commit any changes
 file(COPY_FILE ${CMAKE_CURRENT_BINARY_DIR}/package-lock.json ${CMAKE_CURRENT_SOURCE_DIR}/package-lock.json)
 
@@ -166,6 +150,7 @@ add_custom_target(update_lockfiles
 
 add_copy_file(PackageFiles ${CMAKE_CURRENT_SOURCE_DIR}/README.md ${CMAKE_CURRENT_BINARY_DIR}/README.md)
 add_copy_file(PackageFiles ${CMAKE_CURRENT_SOURCE_DIR}/CHANGELOG.md ${CMAKE_CURRENT_BINARY_DIR}/CHANGELOG.md)
+add_copy_file(PackageFiles ${CMAKE_CURRENT_SOURCE_DIR}/LICENSE ${CMAKE_CURRENT_BINARY_DIR}/LICENSE)
 add_copy_file(PackageFiles ${CMAKE_CURRENT_SOURCE_DIR}/images/lldb.png ${CMAKE_CURRENT_BINARY_DIR}/images/lldb.png)
 add_copy_file(PackageFiles ${CMAKE_CURRENT_SOURCE_DIR}/images/user.svg ${CMAKE_CURRENT_BINARY_DIR}/images/user.svg)
 add_copy_file(PackageFiles ${CMAKE_CURRENT_SOURCE_DIR}/images/users.svg ${CMAKE_CURRENT_BINARY_DIR}/images/users.svg)
@@ -182,6 +167,7 @@ add_custom_target(dev_debugging
 set(PackagedFilesBootstrap
     README.md
     CHANGELOG.md
+    LICENSE
     extension.js
     images/*
     syntaxes/*
